### ISSUE TITLE
multiple "refresh_token"s in ./well-known/openid-configuration payload

### ISSUE THREAD

**Issue by aczerepinski** [doorkeeper-gem/doorkeeper-openid_connect#130]
My payload contains this:

```
"grant_types_supported": [
"authorization_code",
"refresh_token",
"refresh_token",
"refresh_token",
"refresh_token",
... etc (over 30 refresh_token duplicates in the array)
```

Edit to add more information: I have not yet been able to reproduce the bug locally, but on my heroku deployment, a duplicate entry is added to the array every time the page is refreshed. Restarting the dynos resets it. 🤔

**Comment by syakovyn** [comment]
https://github.com/doorkeeper-gem/doorkeeper-openid_connect/commit/328854fe6db48df23433d4c039d5315c8088b227#diff-64410b5830c8849dd99eee50f8779676352aaeaaefcd91e03e52b2217ba0c964 fixes the issue. It is just not yet relased. https://github.com/doorkeeper-gem/doorkeeper-openid_connect/blob/v1.7.4/app/controllers/doorkeeper/openid_connect/discovery_controller.rb doesn't have it.

**Comment by aczerepinski** [comment]
Thank you. That explains why I couldn't reproduce the problem locally - I was running the code at master with the fix already included.

**Comment by toupeira** [comment]
Thanks all, looks like this was fixed in 1.7.5 so I'll go ahead and close this issue!
